jobs/templatetags/jobs_tags.py

```diff
@@ -0,0 +1,7 @@
+from django import template
+
+register = template.Library()
+
+@register.filter(name='addclass')
+def addclass(value, arg):
+    return value.as_widget(attrs={'class': arg})
```

Comment:
Would this be better added with the form creation?  Kind of torn - this is more DRY since you only have to use it once in the form for all fields, but then again, it's custom code for something django already does.  You could do something like this in the form to set the widget attrs for all fields http://stackoverflow.com/a/3800247
If you do keep with the template filter, I think addclass will override any existing 'class' attr that may have been set on a field's widget in the form creation.  This might not be what you want (if it is, maybe the filter would be better named something like 'set_css_classes'?)